org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/GuardedPattern.java

```diff
@@ -193,9 +193,10 @@ public static List propertyDescriptors(int apiLevel, boolean previewEnabled) {
 
 	/**
 	 * Returns the conditional expression of this pattern, or
-	 * <code>null</code> if there is none (the "default:" case).
+	 * <code>this.ast.newNullLiteral()</code> if there is none (the "default:" case).
 	 *
-	 * @return the expression node, or <code>null</code> if there is none
+	 * @return the expression node, or <code>this.ast.newNullLiteral()</code> if there is none
```

Comment:
Please replace this.ast.newNullLiteral() - check other nodes and see what those javadocs say
eg - look at MethodInvocation.getName() 
Please check other places too subsequently - not putting comment